ReactSkia/views/common/RSkImageCacheManager.h

```diff
@@ -10,32 +10,41 @@
 #include "include/core/SkGraphics.h"
 #include "include/gpu/GrDirectContext.h"
 
+#include "ReactSkia/sdk/FollyTimer.h"
+
 #define SKIA_CPU_IMAGE_CACHE_LIMIT  50*1024*1024 // 52,428,800 bytes
 #define SKIA_GPU_IMAGE_CACHE_LIMIT  50*1024*1024 // 52,428,800 bytes
 
 using namespace std;
-
+using namespace rns::sdk;
 namespace facebook {
 namespace react {
 
-typedef facebook::better::map <string,sk_sp<SkImage>> ImageCacheMap;
+typedef struct imageDataExpiryTime {
+  sk_sp<SkImage> imageData;
+  double expiryTime;
+}imageDataExpiryTime;
+
+typedef facebook::better::map <string,imageDataExpiryTime> ImageCacheMap;
```

Comment:
we may not need facebook , since we are already in facebook namespace here

Reply:
yes not need facebook.